packages/react/src/components/Table/TableHead/TableHead.jsx

```diff
@@ -544,6 +553,7 @@ const TableHead = ({
                 truncateCellText={truncateCellText}
                 allowTooltip={false}
                 tooltip={matchingColumnMeta.tooltip}
+                align={isLast ? lastTooltipAlign : align}
```

Comment:
I think we can use something like 
`tooltipDirection={flipTooltipDirection ? 'start' : undefined}`

Reply:
> I think we can use something like `tooltipDirection={flipTooltipDirection ? 'start' : undefined}`

Hello Bjorn,

One question here. Do we allow customizing the tooltip alignment? Previously, it is always set as default which is `start` I  think (we did not pass in the align prop). If so, then I'll only focus on the `rightmostColumn`. Otherwise, we probably need flip the `leftmostColumn`  tooltip if `align` is set to `end`.

Reply:
For the tooltip in the Table column headers there is no customisation, just the default.